capabilityquerier/pkg/capabilityquerier/capabilityquerier.go

```diff
@@ -47,11 +50,24 @@ func GetAndSendCapabilityStatement(
 	queueName string) error {
 
 	var err error
+	metadataURL := &url.URL{}
+	*metadataURL = *fhirURL
+	metadataURL.Path = path.Join(metadataURL.Path, "metadata")
+
+	wellKnownURL := &url.URL{}
+	*wellKnownURL = *fhirURL
+	wellKnownURL.Path = path.Join(wellKnownURL.Path, "/.well-known/smart-configuration")
+
 	message := Message{
 		URL: fhirURL.String(),
 	}
-
-	err = requestCapabilityStatement(ctx, fhirURL, client, &message)
+	// Query fhir endpoint
+	err = requestCapabilityStatement(ctx, metadataURL, "metadata", client, &message)
+	if err != nil {
+		message.Err = err.Error()
+	}
+	// Query well known endpoint
+	err = requestCapabilityStatement(ctx, wellKnownURL, "well-known", client, &message)
```

Comment:
i recommend changing the name of this function as it now queries both the capability statement and the smart-on-fhir well-known endpoint.